core/arch/arm/plat-vexpress/pm_a32.S

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2018, Linaro Limited
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <asm.S>
+#include <arm.h>
+#include <arm32_macros.S>
+#include <kernel/unwind.h>
+#include <platform_config.h>
+
+#if (CFG_TEE_CORE_NB_CORE > 1)
+/*
+ * Current Arm trusted firmware for the AArch32 paltforms traps the secondary
```

Comment:
s/paltforms/platforms/

Reply:
Why ARM-TF switches to monitor mode is another question. I guess I missed that discussion.

Reply:
It is the generic entry in the arm-tf bl1, which applies to primary and secondary boot cores. Arm-tf could be changed. Or Arm-tf/qemu could fix that from the aarch32 hold in pen sequence. Or next stage (i.e optee) can be aware of this and set and core in the expected mode.

Reply:
OK, I understand now. So on primary core ARM-TF enters OP-TEE in secure SVC mode, but the secondaries does so in MON mode. This isn't very clean and just a consequence of the internal design of ARM-TF which shouldn't affect anything external to ARM-TF.
I think ARM-TF should be updated to enter OP-TEE in secure SVC mode also with the secondaries.

Reply:
:-)  I guess we can update the arm-tf plat-qemu to integrate the fix. I'll push for that.
With latest 1.5 we can boot qemu but we need to update the cpu mode from optee. Are you ok to merge this workaournd in op-tee and deprecate this weird way in a later op-tee release, once there'll be an a-tf release tag that includes the changes?

Reply:
Yes